topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,462 @@
+1. LotuS2 on data 8, data 6, and others: main log file
+2. LotuS2 on data 8, data 6, and others: mapping file
+3. LotuS2 on data 8, data 6, and others: Newick-formatted phylogenetic trees between sequences
+4. LotuS2 on data 8, data 6, and others: FASTA-formatted extended OTU seed sequences
+5. LotuS2 on data 8, data 6, and others: OTU abundance matrix
+6. LotuS2 on data 8, data 6, and others: Complete LotuS2 output
+
+> <hands-on-title>Examine outputs</hands-on-title>
+> Click on the little {% icon galaxy-eye %} (eye) icon next to each output in the history to examine it. An explanation of each file is given below.
+>
+> 1. **main log file**: If we had run LotuS on the command line, the contents of the `output/LotuSLogS/LotuS_run.log` would be this main LotuS2 run log file.
+> 
+>     It has information on all the parameters actually passed to the tool, and information on the start time of each step in the program, plus the output of each step.
+>   
+>     You should always check this file first to see if LotuS2 completed correctly, and approximately how many reads were used/classified. If this number is much lower than what you expected, that might indicate a problem with the run and the parameters.
+> 
+> 2. **mapping file**: You can ignore this file as it is a repeat of the mapping tsv file that we used as an input.
+> 
+> 3. **Newick-formatted phylogenetic trees between sequences**: A phylogenetic tree created from the ITS2 sequences. If you want to see what the tree looks like, you can copy the contents of this file and paste it at this [online tree viewer](http://etetoolkit.org/)treeview/
+> 
+> 4. **FASTA-formatted extended OTU seed sequences**: OTU sequences created by the LotuS2 program after clustering near-identical reads
+> 
+> 5. **OTU abundance matrix**: A columnar file with tab-separated-values. Each row is an OTU. The first column has the OTU name, and the remaining columns have the OTU abundance (i.e. how many reads were seen for that OTU) in each sample
+> 
+> 6. **Complete LotuS2 output**: If you try to view this file you will see some unreadable binary characters on the screen. That's because this is a zip file with the complete LotuS2 output folder in one zip folder. You should download this zip file and unzip it on your local computer if you want to see everything inside this folder. ![screenshot of how to download the zip file](images/history-download-zip.png)
+> 
+>     One of the most useful files in this output folder is the `phyloseq.Rdata` file which you can load in R to do further ecological analysis.
+{: .hands_on}
+
+> <question-title>Test your understanding of the outputs</question-title>
+>
+> If for some reason, your LotuS2 run did not complete, you can look at this [example LotuS2 run](https://usegalaxy.eu/u/sujai_spun_earth/h/identifying-mf-from-its2-sequencing-using-lotus2---tutorial-example-run) that we generated using the steps above.
```

Comment:
not sure if we should promote this, since it will only work on EU, but you could also add the results to zenodo, that would actually make it more simple to add the wf to IWC

Reply:
this can also be added in the header like follows, taking an example from single-ceel:

```
answer_histories:
  - label: "UseGalaxy.eu"
    history: https://usegalaxy.eu/u/j.jakiela/h/generating-a-single-cell-matrix-alevin
    date: 2024-03-22
  - label: "Older Alevin version"
    history: https://humancellatlas.usegalaxy.eu/u/wendi.bacon.training/h/cs1pre-processing-with-alevin---answer-key
    date: 2024-01-01
```

Reply:
@paulzierep I've created a new zenodo record at https://zenodo.org/records/13848439 but I still think it is simplest for people to use the import history feature in this version of the tutorial as the tool only works on usegalaxy.eu for now :-)

@hexylena - didn't know that! Thanks - will add!